### DISCUSSION TITLE
How to cite datasets?

### DISCUSSION THREAD

**Discussion by dt-woods** [uslci-admin/private-uslci-content#17] [Q&A] [answered]
For example, in a report or article, what's the recommended reference to something like the [elci_6_2020](https://github.com/FLCAC-admin/uslci-content/blob/dev/downloads/elci_6_2024%20for%20uslci_q2_2024_final.zip) library?

**Comment by bl-young** [comment, marked as answer]
You specifically want to cite the library? or a database that includes a library?

Some of the repositories include a citation in the repo metadata:
![image](https://github.com/user-attachments/assets/7d5c5c8a-378f-4997-81c1-e4635d5cf039)

In general, we recommend that the version number is included in the citation.
